test/unit/module-create-require-block-other/input.mjs

```diff
@@ -0,0 +1,7 @@
+import * as module from 'node:module';
+
+// This require variable is NOT from module.createRequire, so it should be blocked
```

Comment:
What does "blocked" mean here?

Reply:
Maybe the wrong way to describe it, but I meant that it shouldn't register those calls which is the current behavior. So we analyze the `require` named variables through and only if they are created with `module.createRequire` but not the other random variables named `require`.

Does that make sense? should I re-word it differently?

Reply:
Yea, that makes more sense. I will update the test names and comments for clarity